recipes/tttrlib/build.sh

```
@@ -0,0 +1,26 @@
+ -DCMAKE_SWIG_OUTDIR="$SP_DIR" \
+ -DBUILD_LIBRARY=ON \
+ -DPYTHON_VERSION=$(python -c 'import platform; print(platform.python_version())')\
+ -G Ninja ..
+
+# On some platforms (notably aarch64 with Drone) builds can fail due to
+# running out of memory. If this happens, try the build again; if it
+# still fails, restrict to one core.
+ninja install -k 0 || ninja install -k 0 || ninja install -j 1
+
+# Copy programs to bin
+cd $PREFIX/bin
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)